vendor/modules.txt

```diff
@@ -815,6 +815,7 @@ k8s.io/client-go/applyconfigurations/storage/v1beta1
 k8s.io/client-go/discovery
 k8s.io/client-go/discovery/cached
 k8s.io/client-go/discovery/cached/memory
+k8s.io/client-go/discovery/fake
```

Comment:
How does the size of the package change with the additional 225 files?  Does it take longer to download with all the extras?

Reply:
Looks like it adds about ~1.4 MB to the total size of the repo. We are at 71MB in the vendor files. These days, I'd imagine everyone has fairly high speed networks, so maybe 1 additional second for folks to get this update.